cmd/bucket-handlers.go

```diff
@@ -477,6 +485,15 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 	vars := mux.Vars(r)
 	bucket := vars["bucket"]
 
+	objectLockEnabled := false
+	if _, found := r.Header[textproto.CanonicalMIMEHeaderKey("x-amz-bucket-object-lock-enabled")]; found {
```

Comment:
you need to just use `http.CanonicalHeaderKey` not the textproto while the former function is a wrapper it is better to use the http method here.

Reply:
done